apps/omg/lib/omg/typed_data_hash.ex

```diff
@@ -47,6 +47,7 @@ defmodule OMG.TypedDataHash do
   @spec hash_transaction(Transaction.Payment.t()) :: Crypto.hash_t()
   def hash_transaction(%Transaction.Payment{} = raw_tx) do
```

Comment:
this might be stupid but now that we're talking about specification docs so much. could you point me to where did you figure out that a payment transaction's hash consist of these fields? is there a doc or did you read solidity code?

Reply:
https://github.com/omisego/plasma-contracts/blob/master/plasma_framework/contracts/src/transactions/eip712Libs/PaymentEip712Lib.sol#L133 or https://github.com/omisego/plasma-contracts/blob/master/plasma_framework/test/helpers/paymentEip712.js#L32